src/Record.js

```diff
@@ -120,7 +120,11 @@ export class Record {
 
   equals(other) {
     return (
-      this === other || (other && recordSeq(this).equals(recordSeq(other)))
+      this === other ||
+      (other &&
+        this._keys &&
```

Comment:
`this._keys` should always be set because the instance doing the comparison is a Record

Reply:
Will fix